### PR TITLE
Removed Tag buttons for TestCase and TestRun in TestPlan view

### PR DESCRIPTION
**Pull request by asankov**

### CODE REVIEW

File: tcms/templates/plan/get_cases.html [hunk 1]
```diff
@@ -113,16 +94,6 @@
...
-		<div class="mixbar fixed">
-			<div class="tit" style="width: 50px; float: left; height: 100%; margin: 0px; text-align: center;">Tags:</div>
-			<div style="margin-left: 50px; line-height:22px;" class="taglist">
-				{% for tag in case_own_tags %}
-				<a href="#testcases">{{ tag.name }}</a>&nbsp;&nbsp;<span class="grey">-</span>&nbsp;&nbsp;
-				{% empty %}
-				<span class="grey">No tags in cases</span>
-				{% endfor %}
-			</div>
-		</div>
```
Comment (atodorov) [inline tcms/templates/plan/get_cases.html]: This entire block needs to stay. It shows all the tags (as a list) from all of the test cases currently shown.

File: tcms/templates/plan/get_review_cases.html [hunk 2]
```diff
@@ -98,16 +79,6 @@
...
-		<div class="mixbar fixed">
-			<div class="tit" style="width: 50px; float: left; height: 100%; margin: 0px; text-align: center;">Tags:</div >
-			<div  style="margin-left: 50px; line-height:22px;" class="taglist">
-				{% for tag in case_own_tags %}
-				<a href="#testcases">{{ tag.name }}</a>&nbsp;&nbsp;<span class="grey">-</span>&nbsp;&nbsp;
-				{% empty %}
-				<span class="grey">No tags in cases</span>
-				{% endfor %}
-			</div >
-		</div>
```
Comment (atodorov) [inline tcms/templates/plan/get_review_cases.html]: Same as above. This stays for now.

File: tcms/templates/plan/get_cases.html [hunk 3]
```diff
@@ -215,27 +186,3 @@
 	</span>
 	<span class="js-nomore-msg" style="display:none">All cases have been loaded.</span>
 </div>
-{% verbatim %}
-	<script id="batch_tag_form_template" type="text/x-handlebars-template">
-		<form id="id_batch_tag_form">
-			<div class="dia_title" style=" margin:30px 20px;">Please type tag name:</div>
-			<div class="dia_content" style=" margin:30px 20px;">
```
Comment (atodorov) [inline tcms/templates/plan/get_cases.html]: dia_title and dia_content classes are not used anywhere else. Remove from CSS.

File: tcms/templates/plan/get_cases.html [hunk 4]
```diff
@@ -215,27 +186,3 @@
...
-				<input type="text" id="add_tag_plan" name="tags" style="width:300px; height:25px; border:solid  1px #ccc"/>
-				<div id="id_batch_add_tags_autocomplete"></div>
```
Comment (atodorov) [inline tcms/templates/plan/get_cases.html]: Remove styles for id_batch_add_tags_autocomplete from CSS.

File: tcms/templates/plan/get_cases.html [hunk 5]
```diff
@@ -215,27 +186,3 @@
...
-			</div>
-			<div style=" margin:30px 20px;">
-				<input type="submit" value="Submit">
-				<input type="button" value="Cancel" class="js-cancel-button" >
-			</div>
-		</form>
-	</script>
-	<script id="batch_tag_summary_template" type="text/x-handlebars-template">
-		<div class="dia_title" style=" margin:10px 20px;">You have successfully updated tags for the following case(s):</div>
-		<div class="dialog_content">
```
Comment (atodorov) [inline tcms/templates/plan/get_cases.html]: dialog_content class isn't used anywhere else. Remove from CSS.

File: tcms/templates/plan/get_cases.html [hunk 6]
```diff
@@ -215,27 +186,3 @@
...
-			{{# each tags }}
-			<div class="dia_content" style=" margin:10px 20px;">{{ this.pk }} &nbsp; {{ this.fields.summary }}</div>
-			{{/ each }}
-		</div>
-		<input class="dia_btn_close sprites js-close-button" type="button" value="Close" style=" margin:10px 20px;"/>
```
Comment (atodorov) [inline tcms/templates/plan/get_cases.html]: dia_btn_close is not used anywhere else. Remove it from CSS.

File: tcms/static/js/testplan_actions.js [hunk 7]
```diff
@@ -1644,115 +1644,6 @@ function onTestCaseTagFormSubmitClick(options) {
   };
 }
 
-function onTestCaseTagAddClick(options) {
```
Comment (atodorov) [inline tcms/static/js/testplan_actions.js]: Also remove `onTestCaseTagFormSubmit` and `constructBatchtagProcessDialog` functions from this file.

File: tcms/static/js/testcase_actions.js [hunk 8]
```diff
@@ -913,44 +913,6 @@ function removePlanFromCase(container, plan_id, case_id) {
   jQ('#plan_count').text(jQ('table#testplans_table').attr('count'));
 }
 
-function renderTagForm(container, parameters, form_observe) {
```
Comment (atodorov) [inline tcms/static/js/testcase_actions.js]: Also remove `updateCaseTag` from this file.

Comment (asankov) [reply tcms/static/js/testcase_actions.js]: Done

File: tcms/templates/plan/get_cases.html (hunk 3 above)
Comment (asankov) [reply tcms/templates/plan/get_cases.html]: Done

File: tcms/templates/plan/get_cases.html (hunk 4 above)
Comment (asankov) [reply tcms/templates/plan/get_cases.html]: Done

File: tcms/templates/plan/get_cases.html (hunk 5 above)
Comment (asankov) [reply tcms/templates/plan/get_cases.html]: Done

File: tcms/templates/plan/get_cases.html (hunk 6 above)
Comment (asankov) [reply tcms/templates/plan/get_cases.html]: Done

**Pull request merged**